pkg/windowsdriver/procmon/procmon.go

```diff
@@ -73,7 +79,13 @@ func NewWinProcMon(onStart chan *ProcessStartNotification, onStop chan *ProcessS
 	if err := driver.StartDriverService(driverName); err != nil {
 		return nil, err
 	}
-	reader, err := olreader.NewOverlappedReader(wp, procmonReceiveSize, procmonNumBufs)
+	if bufsize == 0 {
+		bufsize = procmonReceiveSize
+	}
+	if numbufs == 0 {
+		numbufs = procmonNumBufs
+	}
+	reader, err := olreader.NewOverlappedReader(wp, bufsize, numbufs)
```

Comment:
question: is there a minimum and maximum for `bufsize`, `numbufs`, and the combination of the 2? If so, then I think we should guard against values that are less than the min and more than the max here before we create the reader. If the values are invalid, then we should set them to their defaults. Depending on if there is a relationship between the 2, we might need another check (e.g. if `bufsize * numbufs` needs to be less than a certain size. If we decide to export the consts as defaults, then the variable names will change, but I am thinking something like:

```suggestion
    if bufsize < minBufSize || bufsize > maxBufSize {
        bufsize = procmonReceiveSize
    }

    if numbufs < minNumBufs || bufsize > maxNumBufs {
        numbufs = procmonNumBufs
    }
    // then, if needed, check the product of the 2 parametes

	reader, err := olreader.NewOverlappedReader(wp, bufsize, numbufs)
```

Reply:
there's no hard & fast limit.  Like, it shouldn't be zero for either.  but beyond that there's nothing really constraining it